src/main/java/org/jabref/gui/preferences/EntryEditorPrefsTab.java

```diff
@@ -114,6 +122,21 @@ public EntryEditorPrefsTab(JabRefPreferences prefs) {
         builder.add(firstNameModeAbbr,  1, 20);
         builder.add(firstNameModeFull, 1, 21);
         builder.add(firstNameModeBoth,  1, 22);
+
+        final ToggleGroup group = new ToggleGroup();
+        Label linkFileOptions = new Label(Localization.lang("Options to add file"));
```

Comment:
Maybe reformulate to `Default drag & drop action`?

Reply:
Fixed as you suggested: https://github.com/JabRef/jabref/pull/4408/files#diff-e0f0e1ade7c4cee8c43b0bb2254c21ebR125